### ISSUE TITLE
Ubuntu 16.04 systray icon missing

### ISSUE THREAD

**Issue by GustavePate** [pydio/pydio-sync-ui#29]
Fresh install of linux pydio-ui 1.2.7 on xubuntu 16.04.
The systray icon for pydio is missing.

I saw that the sync notification is ok so I tested all default xubuntu and ubuntu icons pack in case you used a default icon for the default status (I tested it all without luck).

My workspace is ugly now :(

![screenshot](https://cloud.githubusercontent.com/assets/3317775/20902198/6067e2ee-bb36-11e6-84df-fa14d39ee562.png)

**Comment by 7omate** [comment]
Do you have other apps working with icons (Dropbox / Skype something like that)?

I'd be happy to help you fix it if you're so inclined, but for us as long as it's working on default Ubuntu (and Debian) it's enough.

**Comment by GustavePate** [comment]
Thanks for you help !

In the tray I just have clementine running, nothing special (no skype nor dropbox) killing clementine doesn't change anything...

**Comment by 7omate** [comment]
That's not what I meant. The question was more something in the lines of "Is PydioSync the only application that malfunctions?"

The same code works on debian and ubuntu.

Did you look into https://launchpad.net/sni-qt ?

**Comment by GustavePate** [comment]
Yes, absolutly this is the only "mal" functioning indicator in my systray.
I installed  nextcloud today, work like a charm ;)

Clearly it's the icon that is used for the primary status indictor which is
unknown on all my theme.
After a sucessfull sync i can see in the systray the kind of blue info
bubble that notify it.
But when there's no "special" activity the indicator falls back on the
"normal" icon that is missing or unrecheable on my system.




2016-12-07 18:04 GMT+01:00 7omate <notifications@github.com>:

**Comment by 7omate** [comment]
Indeed it's broken.
It could be linked to the libgstreamer version. How deep are you willing to dig into this issue?

Basically recompiling pydio-sync-ui with a correct version of Qt should work (5.7 is not the correct one, maybe 5.5, 5.6).

We are in the process of moving to Qt 5.7 but it's not ready yet.

**Comment by GustavePate** [comment]
Thanks you 7omate for you analysis.
I'm not in for a re-compilation of the client and the installation of old
qt libs that goes with it.

I will re-try your software later then.

Thanks for all !

2016-12-08 11:52 GMT+01:00 7omate <notifications@github.com>:

**Comment by casasfernando** [comment]
I had the same problem in Elementary OS 0.4 (based on Ubuntu 16) and I managed to solve it by using the same workaround as with Dropbox indicator.
Try to start the UI with the following env variables:

env XDG_CURRENT_DESKTOP=Unity QT_STYLE_OVERRIDE='' pydio-ui

In my case I got the indicator icon to work fine.
Yet unfortunately due to the color combination of the icon itself, if the panel is black you will only see the red ring but the icon will be there.
